Fix controller RBAC permissions for leader election ConfigMap (#105)
giantswarm/nginx-ingress-controller-app: CHANGELOG.md

## [Unreleased]

### Fixed

> - Fix RBAC permissions for controller, allow get and update locking leader election ConfigMap.

Reviewer: ```suggestion
- Fix RBAC permissions for the controller, allowing "get" and "update" locking leader election ConfigMap.
```

> - Fix RBAC permissions for the controller, allowing "get" and "update" locking leader election ConfigMap.

Author: ```suggestion
- Fix controller RBAC permissions, granting "get" and "update" of leader election ConfigMap lock.
```
